docs/docker-credential-ecr-login.1

```diff
@@ -90,6 +104,11 @@ This command is not implemented and is a no-op.
 .TP 16
 .BR erase
 This command is not implemented and is a no-op.
+.SH BUGS
+Kubernetes users using the credential helper inside containers with a non-root
+user may encounter permission issues described here:
+https://github.com/kubernetes-sigs/external-dns/pull/1185.  You may be able to
+work around this bug by adjusting the Kubernetes \fIsecurityContext\fP.
 .SH COPYRIGHT
 Copyright 2018 Amazon.com, Inc. or its affiliates.  All rights reserved.
```

Comment:
Should this be bumped now that we've hit 2020 or does this date not represent the "present" or last file-changed year? :fireworks: :calendar:

Reply:
Changed to "2018-2020".